includes/app-service-mobile-configure-new-backend.md

```diff
@@ -1,4 +1,25 @@
 
-1. Back in the Mobile App backend, click **Settings** > **Quick start** > your client platform. 
-2. Under **Create a table API**, select Node.js for **Backend language**. Accept the acknowledgment and click **Create TodoItem table**. This creates a new *TodoItem* table in your database. Remember that switching an existing backend to Node.js will overwrite all contents! To create a .NET backend instead, [follow these instructions](../articles/app-service-mobile/app-service-mobile-dotnet-backend-how-to-use-server-sdk.md#create-app).
+1. Click **App Services** > select your Mobile App backend > click **Quickstart** > your client platform (iOS, Android, Xamarin, Cordova).
```

Comment:
Looks like the layout of text & image of this file can be improved. Right now the text paragraph and images are cluttered, adding more space between the lines should make it look nicer. :)